src/test/java/org/apache/commons/csv/CSVFileParserTest.java

```diff
@@ -93,11 +93,8 @@ public void testCSVFile(final File testFile) throws Exception {
             try (final CSVParser parser = CSVParser.parse(new File(BASE, split[0]), Charset.defaultCharset(), format)) {
                 for (final CSVRecord record : parser) {
                     String parsed = Arrays.toString(record.values());
-                    if (checkComments) {
-                        final String comment = record.getComment().replace("\n", "\\n");
-                        if (comment != null) {
```

Comment:
Let's keep the caching of "record.getComment()"

Reply:
thanks for review , I have fix that